File: CSharp/k05_1.cs
using System;

namespace CSharp
{
    public class K05_1 : IExecutable
    {
        public void Execute()
        {
            float salary = 19.0f;
            int age = 22;

            while (salary < 50.0f)
            {
                salary *= 1.035f;
                age++;
            }

            Console.Write(age + "歳で月給" + salary + "万円");
        }
    }
}

File: Java/src/com/ecc/gkgo1a/K05_1.java
package com.ecc.gkgo1a;

public class K05_1 implements IExecutable {

    @Override
    public void Execute() {
        float salary = 19.0f;
        int age = 22;

        while (salary < 50.0f)
        {
            salary *= 1.035f;
            age++;
        }

        System.out.print(age + "歳で月給" + salary + "万円");
    }

}
